processor.go

```diff
@@ -612,7 +612,7 @@ func (g *Processor) Setup(session sarama.ConsumerGroupSession) error {
 		// create partition processor for our partition
 		pproc, err := g.createPartitionProcessor(session.Context(), partition, runModeActive,
 			func(msg *message, metadata string) {
-				session.MarkOffset(msg.topic, msg.partition, msg.offset, metadata)
+				session.MarkOffset(msg.topic, msg.partition, msg.offset+1, metadata)
```

Comment:
Please leave a comment why a +1 is required here. It look very arbitrary to the average user.

Reply:
FYI: [tester](https://github.com/lovoo/goka/blob/0085fa7f7c2e542e740ec565af7662b2d441b019/tester/consumergroup.go#L262) vs. [sarama](https://github.com/Shopify/sarama/blob/master/consumer_group.go#L642)
```
// tester
func (cgs *cgSession) MarkMessage(msg *sarama.ConsumerMessage, metadata string) {
	cgs.MarkOffset(msg.Topic, msg.Partition, msg.Offset, metadata)
}
```
```
// sarama
func (s *consumerGroupSession) MarkMessage(msg *ConsumerMessage, metadata string) {
	s.MarkOffset(msg.Topic, msg.Partition, msg.Offset+1, metadata)
}
```
:thinking:

Reply:
Maybe we should use MarkMessage instead of MarkOffset?

Reply:
oh good point regarding the tester. However we cannot use MarkMessage, because the whole point of the original change was to get rid of the sarama-message and have an own message type. But MarkMessage requires the sarama-message

Reply:
and a comment is definitely necessary, will do!

Reply:
In the comment, you may just refer to the sarama code that we are adapting here. Maybe they have a similar issue/discussion/PR where this is already explained?